api/v1/cluster_webhook.go

```diff
@@ -864,6 +864,13 @@ func (r *Cluster) validateConfiguration() field.ErrorList {
 		// validateImageName function
 		return result
 	}
+	if psqlVersion <= 100000 {
```

Comment:
there is a problem here. E.g. version 10.23 will be read as 100023
So we really need to check against 110000